src/components/MapViewer/MenuWidget.jsx

```diff
@@ -282,6 +297,13 @@ export const AddCartItem = ({
                   )}
                 </ul>
               )}
+              {areaData && dataset.IsTimeSeries && (
+                <p>
+                  If you would like to download data for your area of interest
+                  and for the selected time interval, please follow this{' '}
+                  <a href={dataset.DatasetURL + '/download-by-area'}>link</a>.
```

Comment:
We are using <UniversalLink> component from @plone/volto where we show links. This way we can control to open them in a new window if they are external links. See https://github.com/eea/volto-clms-theme/commit/f506ab6089ad1b3da3fe9b12e6b2a7b9a2169023#diff-9dcdae27b377bdb9fe5a3ceb30ebb6ec594968f568d404905308093bceacb3b1R166 as an example